src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner.vue

```diff
@@ -1,80 +1,105 @@
 <template>
-	<div class="sitewide-appeal-wrapper">
-		<div class="sitewide-appeal row" @click="toggleAccordion">
-			<div class="sitewide-header small-12 medium-9 medium-offset-2 large-9 large-offset-2  columns">
-				<h2>
-					<span v-html="bannerHeadline" class="strong"></span>
-					<kv-icon
-						@click="toggleAccordion"
-						:class="{ flipped: open }"
-						class="toggle-arrow"
-						name="medium-chevron"
-						:from-sprite="true"
-					/>
-				</h2>
-			</div>
+	<div ref="pageOverlay">
```

Comment:
Could be something like `:class="{ 'show-overlay': isActive }"` more details below...